cylc/flow/wallclock.py

```diff
@@ -133,46 +136,20 @@ def get_time_string(date_time, display_sub_seconds=False,
     most useful for filenames where ":" may cause problems.
     date_time_is_local - a boolean that, if True, indicates that
     the date_time argument object is in the local time zone, not UTC.
-    custom_time_zone_info (default None) - a dictionary that enforces
-    a particular time zone. It looks like {"hours": _hours,
-    "minutes": _minutes, "string": _string} where _hours and _minutes
-    are the hours and minutes offset from UTC and _string is the string
-    to use as the time zone designator.
 
     """
-    time_zone_string = None
-    if custom_time_zone_info is not None:
-        custom_hours = custom_time_zone_info["hours"]
-        custom_minutes = custom_time_zone_info["minutes"]
-        if use_basic_format:
-            custom_string = custom_time_zone_info["string_basic"]
-        else:
-            custom_string = custom_time_zone_info["string_extended"]
-        if date_time_is_local:
-            date_time_hours = TIME_ZONE_LOCAL_UTC_OFFSET_HOURS
-            date_time_minutes = TIME_ZONE_LOCAL_UTC_OFFSET_MINUTES
-        else:
-            date_time_hours, date_time_minutes = (0, 0)
-        diff_hours = custom_hours - date_time_hours
-        diff_minutes = custom_minutes - date_time_minutes
-        date_time = date_time + timedelta(
-            hours=diff_hours, minutes=diff_minutes)
-        time_zone_string = custom_string
```

Comment:
> @hjoliver, this PR removes a code block that isn't used any more. Can you have a quick look at it and merge if ok.

This code block was previously only used by the Cylc 7 GUI (7.8.15):

https://github.com/cylc/cylc-flow/blob/e60e8f7a6d77256af0e312237c6c8f7bf4fd38f8/lib/cylc/gui/updater_tree.py#L269-L277

However, it looks like it was only used for the same purpose that is already provided by `override_use_utc`:

https://github.com/cylc/cylc-flow/blob/e60e8f7a6d77256af0e312237c6c8f7bf4fd38f8/lib/cylc/gui/updater_tree.py#L190

https://github.com/cylc/cylc-flow/blob/e60e8f7a6d77256af0e312237c6c8f7bf4fd38f8/lib/cylc/state_summary_mgr.py#L125-L128

Reply:
@hjoliver poke